src/haiway/files/__init__.py

```
@@ -0,0 +1,30 @@
+from haiway.files.config import FileSystemConfig
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)